docs/en/how-to_guides/ThanoSQL_connecting/rest/rest_api_thanosql_file.md

```diff
@@ -4,20 +4,11 @@ title: File APIs
 
 # **File APIs**
 
-You can use File APIs to remotely send and upload files to your Workspace storage and insert them into any of your table within the database. The default root folder for all user files is set to "drive."
-
-!!! warning "__Warning__"
-    - File API supports image, audio, and video data with the following extensions:
-        - Image:  "jpg", "png", "jpeg"
-        - Audio: "mp3", "wav
-        - Video: "mp4", "wmv", "avi"
-        
-    - Files with extensions not listed above are stored in the "drive/others" folder.
-
+You can use File APIs to remotely send and upload files to your Workspace storage and insert them into any of your table within the database. The default root folder for all user files uploaded by this APIs are set to 'drive/'.
 
 ## __`GET` /file/__
 
-A list of files and folders is returned from a specified file path. The file path can be expressed using a regular expression.
+Returns a list of files and folders that match the given path. The path shoulde include root foolder 'drive/' and it can be expressed using a regular expression.
```

Comment:
```suggestion
Returns a list of files and folders that match the given path. The path should include root folder 'drive/' and it can be expressed using a regular expression.
```